core/src/main/protobuf/org/apache/spark/status/protobuf/store_types.proto

```diff
@@ -100,11 +100,21 @@ message TaskDataWrapper {
   int64 shuffle_remote_bytes_read_to_disk = 34;
   int64 shuffle_local_bytes_read = 35;
   int64 shuffle_records_read = 36;
-  int64 shuffle_bytes_written = 37;
-  int64 shuffle_write_time = 38;
-  int64 shuffle_records_written = 39;
-  int64 stage_id = 40;
-  int32 stage_attempt_id = 41;
+  int64 shuffle_corrupt_merged_block_chunks = 37;
+  int64 shuffle_merged_fetch_fallback_count = 38;
+  int64 shuffle_merged_remote_blocks_fetched = 39;
+  int64 shuffle_merged_local_blocks_fetched = 40;
+  int64 shuffle_merged_remote_chunks_fetched = 41;
+  int64 shuffle_merged_local_chunks_fetched = 42;
+  int64 shuffle_merged_remote_bytes_read = 43;
+  int64 shuffle_merged_local_bytes_read = 44;
+  int64 shuffle_remote_reqs_duration = 45;
+  int64 shuffle_merged_remote_req_duration = 46;
+  int64 shuffle_bytes_written = 47;
+  int64 shuffle_write_time = 48;
+  int64 shuffle_records_written = 49;
+  int64 stage_id = 50;
+  int32 stage_attempt_id = 51;
```

Comment:
We should not change attribute `id`'s (here and elsewhere)
In this specific case, given this is not yet released, it is not as bad - but it might still interfere with some ongoing dev/test work.

Reply:
@mridulm Do you suggest we keep the attribute `id`s same as before ? This would mean having to change the function definitions across most of the diff in this PR

Reply:
Protobuf id's should never be changed.

Reply:
@mridulm Thanks for reviewing, changed it to keep the existing protobuf ids